lib/kafka/client.rb

```diff
@@ -414,6 +414,32 @@ def last_offset_for(topic, partition)
       @cluster.resolve_offset(topic, partition, :latest) - 1
     end
 
+
+    # Retrieve the offset of the last message in each partition of the specfied topics.
+    #
+    # @param topics [String, Array<String>] single topic name or array of topic names.
+    #   nil means all topics will be fetched.
+    #
+    # @return [Hash]
+    #
+    # @example
+    #   last_offsets_for ['topic-1', 'topic-2']  ->
+    #   {
+    #     'topic-1' => { 0 => 100, 1 => 100 },
+    #     'topic-2' => { 0 => 100, 1 => 100 },
+    #   }
+    def last_offsets_for(topics = nil)
+      topics   = [topics] if !topics.nil? && topics.is_a?(String)
+      topics ||= self.topics
```

Comment:
Thanks for the quick feedback!
Re: #1, sure, I will change it.
Re: #2, great suggestion!